src/lsptoolshost/roslynLanguageServer.ts

```diff
@@ -673,7 +673,13 @@ function getServerFileName(platformInfo: PlatformInformation) {
         extension = '.dll';
     }
 
-    return `${serverFileName}${extension}`;
+    let pathWithExtension = `${serverFilePath}${extension}`;
+    if (!fs.existsSync(pathWithExtension)) {
```

Comment:
So on Linux we don't set extension, expect this to fail and then take the if block?

Reply:
Oh right, I forgot there'd be no extension. It _might_ make sense to put a comment there that the lack of setting that is intentional, but...